spec.bs

```diff
@@ -3876,7 +3946,11 @@ from querying the server during an auction.
 
 <div algorithm>
   To <dfn>query generated bid k-anonymity count</dfn> given a [=generated bid=] |bid|:
-    1. Compute the |adHashCode| following [=compute the key hash of ad=] with the |bid|'s [=generated bid/interest group=] and |bid|'s [=generated bid/ad descriptor=].
+    1. Let |selectedReportingId| be a [=string=]-or-null set to null

```

Comment:
`|selectedReportingId|` is set but never used. I assume this is supposed to be checking k-anon for reporting if `selectedBuyerAndSellerReportingId` is set? If so, I'd recommend moving this down to the bottom of this algorithm, just before `1. Return true.`:

```
1. If |bid|'s [=generated bid/selected buyer and seller reporting ID=] [=map/exists=]:
  1. Let |reportingHashCode| be the result of [=compute the key hash of reporting ID=] with the |bid|'s [=generated bid/interest group=], the |bid|'s [=generated bid/selected buyer and seller reporting ID=].
  1. If [=query k-anonymity cache=] for |reportingHashCode| returns false, return false.